test: adjustmets to run with f-42

gstarovo opened the pull request:
The test is forked from [the pull request](https://github.com/QUBIP/pq-container/pull/32) to the main branch and is adjusted to the expected outputs of fedora-42: 
- first test:  default connection group: UNKNOWN (4588)
- third test: the searched output is placed in two lines, due to the fact that grep takes input line by line, the needed output is concatenated to be used by grep.

gstarovo commented:
@lsm5 thank you, I will fix that. 

Also, what do you mean by 

> still being reported as success

It shows the overall result as FAIL, that is correct
`::   OVERALL RESULT: FAIL`

lsm5 commented:
> It shows the overall result as FAIL, that is correct `:: OVERALL RESULT: FAIL`

I mean, if the overall result is FAIL, the `testing-farm-*` jobs should have final status as fail. In the previous run, they had a success tick mark against them.

gstarovo commented:
Here and in [rawhide pull request](https://github.com/QUBIP/pq-container/pull/32), it fails to provision the test environment. How can I retry it? Or is it because of my latest changes (because it was not happening before)? I have added .expect file and updated the container file.

lsm5 commented:
> Here and in [rawhide pull request](https://github.com/QUBIP/pq-container/pull/32), it fails to provision the test environment. How can I retry it? Or is it because of my latest changes (because it was not happening before)? I have added .expect file and updated the container file.

I don't think your changes will cause provisioning issues. May have had something to do with the recent Fedora  outage affecting a lot of stuff. I suggest retrying later or maybe next week.

lsm5 commented:
/packit test

:crossed_fingers:

lsm5 commented:
Packit itself is trudging along right now as the Fedora outage has affected it too. So, let's give this a while.

gstarovo commented:
@lsm5, may start the konflux tests when the testing farm tests pass, or only maintainers?

lsm5 commented:
/ok-to-test

sahanaprasad07 commented:
LGTM
